clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -943,41 +943,41 @@ def create_adni_sessions_dict(
     # Iterate over the metadata files
 
     for location in files:
-
         location = location.split("/")[0]
-        if path.exists(path.join(clinical_data_dir, location)):
-
-            file_to_read_path = path.join(clinical_data_dir, location)
-            cprint(f"\tReading clinical data file: {location}")
+        pattern = location.split(".")[0] + "(_\d{1,2}[A-Za-z]{3}\d{4})?.csv"
```

Comment:
Any reason not to use your `load_clinical_csv()` function here ?

Reply:
I must have missed that one

Reply:
Ha yes, actually there was. The way this was built before I changed things was different and I did not see how to use `load_clinical_csv`

Reply:
I'm probably missing something, but I don't understand why you couldn't do:

```python
for location in files:
    location = location.split("/")[0]
    df_file = load_clinical_csv(clinical_data_dir, location.split(".")[0])
    df_filtered = filter_subj_bids(df_file, location, bids_ids).copy()
    ...
```

Reply:
You are right, it looks like this should work

Reply:
I nested this part in a try. I'm not fond of it, but it works. WDYT